stable/admin/README.md

```diff
@@ -114,6 +114,8 @@ The following tables list the configurable parameters for the `nuodb` option:
 | `image.tag` | NuoDB container image tag | `latest` |
 | `image.pullPolicy` | NuoDB container pull policy |`Always`|
 | `image.pullSecrets` | Specify docker-registry secret names as an array | [] (does not add image pull secrets to deployed pods) |
+| `serviceAccount` | The name of the service account used by NuoDB Pods | `nuodb` |
+| `addRoleBinding` | Whether to add role and role-binding giving `serviceAccount` access to Kubernetes APIs (Pods, PersistentVolumes, PersistentVolumeClaims, StatefulSets) | `true` |
```

Comment:
I suggest making this specific to the inspector role binding or make it a list of roles that we would like to attach to NuoDB service account. This way we could extend it in future with all roles needed for the NuoDB pods to run.

Reply:
This is to support functionality that I intend to add to the Admin. I don't expect the end user to care about which APIs the NuoDB has access to; we should just enable all the ones that we use in order to do clean up actions like what I described in the summary.

Reply:
What I meant is that the service account is one and multiple roles can be bind to it. This is why I found `addRoleBinding` not specific enough to the inspector role. This is just a minor suggestion  from user point of view if you decide to change it.